templates/regression/index.html

```diff
@@ -109,5 +112,25 @@ <h5>Regression tests</h5>
             });
             switchCategory();
         });
+        function toggleactive(test_id) {
+            $.ajax({
+                type: "GET",
+                url: '/regression/test/' + test_id + '/toggle',
```

Comment:
Try to use the `url_for`, it's available anyhow... Store the url in a variable with a replacement value, and replace that at runtime with the test_id.